notioncli/cli.py

```diff
@@ -0,0 +1,170 @@
+import functools
+import os
+import argparse
+import sys
+
+from termcolor import colored, cprint
+
+from notion.client import NotionClient
+from notion.block import TodoBlock
+
+from contextlib import redirect_stdout
+from contextlib import contextmanager
+from io import StringIO
+
+@contextmanager
+def capture_stdout(output): 
+    stdout = sys.stdout
+    sys.stdout = output
+    try:
+        yield
+    finally:
+        sys.stdout = stdout
+
+
+class ConfigurationError(Exception):
+    pass
+
+
+def connect():
+    try:
+        client = NotionClient(token_v2=os.environ['NOTION_TOKEN'], monitor=False)
+    except:
+        raise ConfigurationError('Invalid or expired NOTION_TOKEN')
+
+    return client
+
+
+def get_page(client):
+    if "NOTION_PAGE" not in os.environ:
+        raise ConfigurationError('Missing NOTION_PAGE')
+    return client.get_block(os.environ['NOTION_PAGE'])
+
+    
+def parse_task(taskn):
+    taskn = str(taskn)
+    if ',' in string:
+        taskn = string.split(',')
+    return taskn
+
+
+def check_env():
+    has_token = 'NOTION_TOKEN' in os.environ
+    has_page = 'NOTION_PAGE' in os.environ
+
+    if not has_token:
+        if has_page:
+            raise ConfigurationError("Missing NOTION_TOKEN")
+        else:
+            raise ConfigurationError("Missing NOTION_TOKEN and NOTION_PAGE")
+
+def list_tasks(page):
```

Comment:
I renamed these because `list` is a core Python thing and shadowing builtins drives me nutty ;)

Reply:
it's a dangerous game for sure